frontend/src/components/RatingTable.tsx

```diff
@@ -66,16 +74,24 @@ export const ratingTable: (def: RatingTableDef) => FC<RatingTableProps> = ({
   const listRef = useRef<VariableSizeList<any> | null>(null);
   const [divRef, setDivRef] = useState<HTMLDivElement | null>(null);
   const { t } = useTranslation();
+  const [sort, sorting] = useSorting(taskRatingsSort);
   const [scrollBarWidth, setScrollBarWidth] = useState(0);
   const [rowHeights, setRowHeights] = useState<number[]>([]);
   const [listHeight, setListHeight] = useState(0);
   const typeString =
     type === TaskRatingDimension.BusinessValue ? 'value' : 'complexity';
   const roadmapId = useSelector(chosenRoadmapIdSelector);
 
+  const sorted = useMemo(() => sort(ratings), [ratings, sort]);
+
+  useEffect(() => {
+    if (!sorting.type.get())
+      sorting.type.set(TaskRatingSortingTypes.SORT_FOR_CUSTOMER);
+  }, [sorting.type]);
```

Comment:
`if (!sorting.type.get())` is bit risky, as the value can be `0`, not just `undefined`. In this case `0` happens to also be `SORT_FOR_CUSTOMER` but this is waiting for problems.

I'd rather add a default sort ordering parameter for `useSorting`. Something like
```ts
export const useSorting = <SortingType, ItemType, Default extends SortingType>(
  getSort: (t: SortingType | undefined) => SortBy<ItemType>,
  defaultType?: Default,
) => {
  const [type, setType] = useState<SortingType | undefined>(defaultType);
```

This can also be useful in other places, and gets rid of the `useEffect`